tests/integration/test_data/npm_multiple_packages_lockfile3/bom.json

```diff
@@ -73,6 +73,22 @@
       "type": "library",
       "version": "1.1.1"
     },
+    {
+      "name": "assertion-error",
```

Comment:
Do we have any dependency that is also present in first_pkg or second_pkg? It would be a good way to test we're only reporting missing hashes in `third_pkg`.

(`is-positive` appears in all pkgs, but since it is a git dependency, it does not have hashes)

Reply:
added sax as a dependency to `third_pkg`, this one is also in second_pkg